src/controller/org.controller/org.controller.js

```diff
@@ -573,12 +583,19 @@ async function updateUser (req, res, next) {
     }
     if ('active_roles.remove' in req.ctx.query) {
       if (Array.isArray(req.ctx.query['active_roles.remove'])) {
-        req.query['active_roles.remove'].forEach(r => {
+        req.ctx.query['active_roles.remove'].forEach(r => {
```

Comment:
Good catch! Pease fix this (i.e., `req.ctx.query`) in line 337 when updating an Org.